library/Businessprocess/Renderer/TileRenderer/NodeTile.php

```diff
@@ -102,11 +102,7 @@ public function render()
 
             $this->add($link);
         } else {
-            $this->add(Html::tag(
-                'span',
-                ['class' => 'missing-node-msg'],
-                sprintf('Trying to access a missing business process node "%s"', $node->getNodeName())
-            ));
+            $this->add(Html::tag('a', $node->getAlias()));
```

Comment:
Should be clickable. The icons in the top left corner are still active but the node not.

Reply:
fixed